implementations/rust/ockam/ockam_command/src/vault/default.rs

```diff
@@ -29,27 +28,20 @@ impl DefaultCommand {
 fn run_impl(opts: CommandGlobalOpts, cmd: DefaultCommand) -> miette::Result<()> {
     let DefaultCommand { name } = cmd;
     let state = opts.state.vaults;
-    match state.get(&name) {
-        Ok(v) => {
-            // If it exists, warn the user and exit
-            if state.is_default(v.name())? {
-                Err(miette!("Vault '{}' is already the default", name))
-            }
-            // Otherwise, set it as default
-            else {
-                state.set_default(v.name())?;
-                opts.terminal
-                    .stdout()
-                    .plain(fmt_ok!("Vault '{name}' is now the default"))
-                    .machine(&name)
-                    .json(serde_json::json!({ "vault": {"name": name} }))
-                    .write_line()?;
-                Ok(())
-            }
-        }
-        Err(err) => match err {
-            CliStateError::NotFound => Err(miette!("Vault '{}' not found", name)),
-            _ => Err(err.into()),
-        },
+    let v = state.get(&name)?;
+    // If it exists, warn the user and exit
+    if state.is_default(v.name())? {
+        Err(miette!("Vault '{}' is already the default", name))
```

Comment:
Perhaps rephrase with: "Nothing to do, as vault '{}' is already the default"

Reply:
To make it less verbose, you might also consider "The vault '{}' is already the default"